api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1324,6 +1345,55 @@ private HttpResponse executePost(String bucketName, String objectName, Map<Strin
   }
 
 
+  private Map<String, String> amzPrefixHeaderMap(Map<String,String> headerMap) {
```

Comment:
* Have method name as `normalizeHeaders()` and the logic is no more than like below
```java
Map<String,String> normHeaderMap = new HashMap<String,String>();
for (Map.Entry<String, String> entry : headerMap.entrySet()) {
  String key = entry.getKey();
  String value = entry.getValue();
  if (amzHeaders.contains(key)) {
    key = "x-amz-" + key;
  } else if (!standardHeaders.contains(key)) {
    key = "x-amz-meta-" + key;
  }
  normHeaderMap.put(key, value);
}
return normHeaderMap;
```

Reply:
Done